io_export_objex2/properties.py

```diff
@@ -585,10 +640,31 @@ class ObjexMaterialProperties(bpy.types.PropertyGroup):
             default=True
         )
     geometrymode_G_SHADING_SMOOTH = bpy.props.BoolProperty(
-            name='Smooth Shading',
-            description='G_SHADING_SMOOTH\n' 'Enable smooth shading (vertex colors, lighting)',
-            default=True
-        )
+        name='Smooth Shading',
+        description='G_SHADING_SMOOTH\n' 'Enable smooth shading (vertex colors, lighting)',
+        default=True
+    )
+    
+
+    shading = bpy.props.EnumProperty(
+        items=[
+            ('LIGHTING',     'Lighting'     ,''),
+            ('VERTEX_COLOR', 'Vertex Color' ,''),
```

Comment:
This takes the addon in a weird direction because I always intended for the nodes to be the primary way to store information, in the name of wysiwyg.

But clearly nodes aren't as popular as I think they're cool

by making a prop for this, it means the nodes and this prop can go out of sync, if the nodes are modified directly (which is the intended way), then the UI value won't update

The export will follow the nodes, so it ends up being weird idk.